tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2057,7 +2057,19 @@ def test_tiny_timestamp_generation(self):
 
         generated_ids = model.generate(input_features, max_length=448, return_timestamps=True).to("cpu")
 
-        EXPECTED_OUTPUT = torch.tensor([50258, 50259, 50359, 50364, 2221, 13, 2326, 388, 391, 307, 264, 50244, 295, 264, 2808, 5359, 11, 293, 321, 366, 5404, 281, 2928, 702, 14943, 13, 50692, 50692, 6966, 307, 2221, 13, 2326, 388, 391, 311, 9060, 1570, 1880, 813, 702, 1871, 13, 50926, 50926, 634, 5112, 505, 300, 412, 341, 42729, 3196, 295, 264, 1064, 11, 365, 5272, 293, 12904, 9256, 450, 10539, 51208, 51208, 949, 505, 11, 14138, 10117, 490, 3936, 293, 1080, 3542, 5160, 881, 26336, 281, 264, 1575, 13, 51552, 51552, 634, 575, 12525, 22618, 1968, 6144, 35617, 7354, 1292, 6, 589, 307, 534, 10281, 934, 439, 11, 293, 51836, 51836, 50257])  # fmt: skip
+        EXPECTED_OUTPUT = torch.tensor([
+            50258, 50259, 50359, 50364, 2221, 13, 2326, 388, 391, 307, 264, 50244, 295,
+            264, 2808, 5359, 11, 293, 321, 366, 5404, 281, 2928,
+            702, 14943, 13, 50692, 50692, 6966, 307, 2221, 13, 2326,
+            388, 391, 311, 9060, 1570, 1880, 813, 702, 1871, 13,
+            50926, 50926, 634, 5112, 505, 300, 412, 341, 42729, 3196,
+            295, 264, 1064, 11, 365, 5272, 293, 12904, 9256, 450,
+            10539, 51208, 51208, 949, 505, 11, 14138, 10117, 490, 3936,
+            293, 1080, 3542, 5160, 881, 26336, 281, 264, 1575, 13,
+            51552, 51552, 634, 575, 12525, 22618, 1968, 6144, 35617, 7354,
+            1292, 6, 589, 307, 534, 10281, 934, 439, 11, 293,
+            51836, 50364, 393, 4411, 13, 50514, 50257
+        ])
```

Comment:
(nit): Using `# fmt: skip` signals that the code line won't be change when doing `make fixup`

I'm also suggesting that we keep everything into the same line, but that's not an obligation 
```suggestion
        EXPECTED_OUTPUT = torch.tensor([50258, 50259, 50359, 50364, 2221, 13, 2326, 388, 391, 307, 264, 50244, 295, 264, 2808, 5359, 11, 293, 321, 366, 5404, 281, 2928, 702, 14943, 13, 50692, 50692, 6966, 307, 2221, 13, 2326, 388, 391, 311, 9060, 1570, 1880, 813, 702, 1871, 13, 50926, 50926, 634, 5112, 505, 300, 412, 341, 42729, 3196, 295, 264, 1064, 11, 365, 5272, 293, 12904, 9256, 450, 10539, 51208, 51208, 949, 505, 11, 14138, 10117, 490, 3936, 293, 1080, 3542, 5160, 881, 26336, 281, 264, 1575, 13, 51552, 51552, 634, 575, 12525, 22618, 1968, 6144, 35617, 7354, 1292, 6, 589, 307, 534, 10281, 934, 439, 11, 293, 51836, 50364, 393, 4411, 13, 50514, 50257]) # fmt: skip
```

Reply:
Or you can use `# fmt: off` and `# fmt: on` on multiple lines:
```suggestion
        # fmt: off
        EXPECTED_OUTPUT = torch.tensor([
            50258, 50259, 50359, 50364, 2221, 13, 2326, 388, 391, 307, 264, 50244, 295,
            264, 2808, 5359, 11, 293, 321, 366, 5404, 281, 2928,
            702, 14943, 13, 50692, 50692, 6966, 307, 2221, 13, 2326,
            388, 391, 311, 9060, 1570, 1880, 813, 702, 1871, 13,
            50926, 50926, 634, 5112, 505, 300, 412, 341, 42729, 3196,
            295, 264, 1064, 11, 365, 5272, 293, 12904, 9256, 450,
            10539, 51208, 51208, 949, 505, 11, 14138, 10117, 490, 3936,
            293, 1080, 3542, 5160, 881, 26336, 281, 264, 1575, 13,
            51552, 51552, 634, 575, 12525, 22618, 1968, 6144, 35617, 7354,
            1292, 6, 589, 307, 534, 10281, 934, 439, 11, 293,
            51836, 50364, 393, 4411, 13, 50514, 50257
        ])
        # fmt: on
```

Reply:
I prefer the mutiple lines version for improved readability. Thanks for `#fmt: off/ on`, I was not aware of that !